soccer/src/soccer/strategy/coach/coach_node.cpp

```diff
@@ -285,15 +285,20 @@ void CoachNode::assign_positions_normal(std::array<uint32_t, kNumShells>& positi
     //  1. 1 Goalie
     //  2. 2 Offense
     //  3. Remaining Defense (always 1 defender)
+    RCLCPP_INFO(this->get_logger(), "assigning positions"); 
     int assign_num = 0;
     for (u_int8_t robot_id = 0; robot_id < kNumShells; robot_id++) {
+        //RCLCPP_INFO(this->get_logger(), "robot alive: " + check_robot_alive(robot_id));
         if (check_robot_alive(robot_id)) {
+            RCLCPP_INFO(this->get_logger(), "ANSON: Assigning id=%d", robot_id); 
             switch (assign_num) {
                 case 0:
                     positions[robot_id] = Positions::Goalie;
                     break;
                 case 1:
-                    positions[robot_id] = Positions::Offense;
+                    RCLCPP_INFO(this->get_logger(), "ANSON: Assigning Runner! (before)"); 
```

Comment:
make sure to remove extra debug messages before commiting, unless particularly useful for the future (like an Error or Warning)